resources/profiles/Flashforge/process/0.20mm Standard @Flashforge AD5M Pro 0.4 Nozzle.json

```diff
@@ -9,10 +9,10 @@
   "reduce_crossing_wall": "0",
   "max_travel_detour_distance": "0",
   "bottom_surface_pattern": "monotonic",
-  "bottom_shell_layers": "5",
+  "bottom_shell_layers": "3",
   "bottom_shell_thickness": "0",
   "bridge_flow": "1",
-  "bridge_speed": "25",
+  "bridge_speed": "50",
```

Comment:
I suggest to have 25 as default for best bridge performance though

Reply:
Ok, noted. I will revert the change for this value in a new commit. This was done to align with the other Flashforge profiles, should it be updated to 25 for all profiles?

Reply:
I have now updated the bridge speed value to 25 for all profiles.